### PR TITLE
Add new fields to telemetry ingress protobuf

### PR DESCRIPTION
**Pull request by skubakdj**
## What
- Introduces `TelemetryType`s that will be included with telemetry payloads 
- Adds `TelemetryType` field to the ingress protobuf
- Adds `SentAt` field to the ingress protobuf
- Uses a no-op telemetry client in DKG jobs

## Why 
- Including `TelemetryType` in the protobuf allows our ingress service to route the telemetry correctly
- Including `SentAt` in the protobuf allows our ingress service to more accurately measure network latency 
- Our ingress service currently does not support DKG telemetry due to possible size limitations, so a no-op client is being explicitly used

### CODE REVIEW

**Review by essamhassan — DISMISSED** [review]
LGTM, quick not blocking question: Any particular immediate impact to users of telem with this new field? Would this affect aggr queries for example?

File: core/services/synchronization/telemetry_ingress_batch_worker.go [hunk 1]
```diff
@@ -4,10 +4,11 @@ import (
 	"sync"
 	"time"
 
+	"go.uber.org/atomic"
```
Comment (jmank88) [inline core/services/synchronization/telemetry_ingress_batch_worker.go:7]: We just switched all of these to the std lib in #8345

Comment (skubakdj) [reply core/services/synchronization/telemetry_ingress_batch_worker.go:7]: good to know, will update the PR

**Comment by skubakdj** [comment]
@essamhassan this PR only modifies the protobuf wrapper used to send the telemetry payload to our ingress service (which will be upgraded to support these changes). the underlying telemetry payloads are the same, so it shouldn't impact any consumers of the telemetry

**Comment by gheorghestrimtu** [comment]
Based on the e2e test https://github.com/smartcontractkit/monitoring/pull/1414 written for the case of OCR2VRF, the product specific telemetry reaches OTI fine.

**Review by essamhassan — APPROVED** [review]

**Pull request merged**